### DISCUSSION TITLE
Selector by PropertyEquals for Interfaces

### DISCUSSION THREAD

**Discussion by RomanSoloweow** [AvaloniaUI/Avalonia#6273] [Q&A]
I need to perform a selector by property value, for example [IsSelected = true], but all elements with this name are generic classes.

I would like to implement the selector using the ISelectable interface [IsSelected = true], but I cannot do this since the selector requires an AvaloniaProperty.

What are the workarounds?

**Comment by maxkatz6** [comment]
Classes binding. 
Classes.my-class="{Binding MyInterfaceProp}"

**Reply by RomanSoloweow** [reply]
It's work for Generic Classes?

**Reply by maxkatz6** [reply]
Did you try? It should.
